scripts/benchmark_clang_tv.in

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o xtrace # can remove this later, for now it's nice to see commands
+
+# TODO support subsetting the benchmarks from the command line
+
+if [ "$#" -ne 1 ]; then
+    cat << EOM
+Expected one command line argument: the name of an initially-empty working
+directory for sources, binaries, and reports.
+EOM
+    exit -1
+fi
+
+WORK=$1
+
+# TODO give alivecc a mode where it doesn't invoke alive at all, so we
+# can test scripts like this more quickly and easily
+
+FLAGS='-mllvm -tv-parallel-unrestricted -mllvm -tv-disable-undef-input -mllvm -tv-disable-poison-input -mllvm -tv-smt-to=30000 -mllvm -tv-subprocess-timeout=600 -mllvm -tv-overwrite-reports'
```

Comment:
can we keep poison enabled at least? It usually doesn't have such a great impact on performance, and we would better confirm optimizations can handle it.
I know value mismatches are usually more exciting, but we have scripts to filter the output and pick just those.
For undef, well, I'm ok to disable those for now. At least until the memory implementation can handle those more efficiently.